examples/resources/powerstore_volumegroup_snapshot/resource.tf

```diff
@@ -1,11 +1,12 @@
 # Commands to run this tf file : terraform init && terraform plan && terraform apply
 # Create, Update, Delete is supported for this resource
 # To import , check volumegroup_snapshot_import.tf for more info
-# name and volume_group_id/volume_group_name are the required attributes to create and update.
+# name and volume_group_id/volume_group_name are the required attributes to create volume group snapshot.
 # description and expiration_timestamp are the optional attributes.
 # Either volume_group_id or volume_group_name should be present.
 # VolumeGroup DataSource can be used to fetch volume group ID/Name
-# expiration_timestamp if present in config cannot be ""
+# During create operation, if expiration_timestamp is not specified or set to "", snapshot will be created with infinite retention
```

Comment:
can we say `is not specified or set to blank` instead of `is not specified or set to ""` for all such instances.